stl/inc/algorithm

```diff
@@ -3010,6 +3010,65 @@ void replace(_ExPo&& _Exec, const _FwdIt _First, const _FwdIt _Last, const _Ty&
     const _Ty& _Newval) noexcept; // terminates
 #endif // _HAS_CXX17
 
+#ifdef __cpp_lib_concepts
+namespace ranges {
+
+    // VARIABLE ranges::replace
+    class _Replace_fn : private _Not_quite_object {
+    public:
+        using _Not_quite_object::_Not_quite_object;
+
+        // clang-format off
+        template <input_iterator _It, sentinel_for<_It> _Se, class _Ty1, class _Ty2, class _Pj = identity>
+            requires indirectly_writable<_It, const _Ty2&> &&
+                     indirect_binary_predicate<equal_to, projected<_It, _Pj>, const _Ty1*>

```

Comment:
I've been trying to redundantly `ranges::` (or _RANGES for functions) qualify names that exist in both `std` and `std::ranges` to help readers with minimal context. Could we qualify `ranges::equal_to` here?

We conventionally wrap before binary operators in long expressions rather than after (Ugh, I can't wait for clang-format to grok requires-clauses).

These comments apply to 3037 as well.

Reply:
Done